llama_stack/apis/agents/agents.py

```diff
@@ -296,6 +296,9 @@ class AgentTurnCreateRequest(AgentConfigOverridablePerTurn):
     stream: Optional[bool] = False
     tool_config: Optional[ToolConfig] = None
 
+    # TODO (xiyan): used for backward compatibility, update for 0.1.5
```

Comment:
nit: update the comment to say "temporary flag. remove this for 0.1.5"

Reply:
done